README.md

```diff
@@ -1,67 +1,91 @@
-- [Pods](https://aurae.io/stdlib/v0/#pod) (Cells running in spawned instances)
-- [Virtual Machines](https://aurae.io/stdlib/v0/#virtualmachine) (Long-lived arbitrary virtual machines)
-
-### auraed
-
-Think of [auraed](https://github.com/aurae-runtime/aurae/tree/main/auraed) as a PID 1 init machine daemon with a scope similar to [systemd](https://www.freedesktop.org/wiki/Software/systemd/) and functionality similar to [containerd](https://github.com/containerd/containerd) and [firecracker](https://github.com/firecracker-microvm/firecracker).
-
-### Authentication
-
-Aurae brings [SPIFFE](https://github.com/spiffe)/[SPIRE](https://github.com/spiffe/spire) (x509 mTLS) backed identity, authentication (authn) and authorization (authz) as low as the Unix domain socket layer in a distributed system.
-
-### Standard Library
-
-Aurae exposes its functionality over a gRPC API which is referred to as the [Aurae Standard Library](https://aurae.io/stdlib/).
-
-### Principle of Least Awareness
-
-A single Aurae instance has no awareness of higher order scheduling mechanisms such as the Kubernetes control plane. Aurae is designed to take ownership of a single node, and expose the standard library as a generic and meaningful way for higher order consumers.
-
-Aurae is a low level building block and is designed to work well with any higher order system by offering a thoughtful set of APIs and controls for managing workloads on a node.
-
-### Motivation
-
-Read [Why fix Kubernetes and Systemd](https://medium.com/@kris-nova/why-fix-kubernetes-and-systemd-782840e50104) by [Kris Nóva](https://github.com/krisnova).
-
-Aurae attempts to simplify and improve the stack in enterprise distributed systems by carving out a small portion of responsibility while offering a few basic guarantees with regard to state, synchronicity, awareness, and security.
-
-Aurae brings enterprise identity as low as the socket layer in a system, which unlocks multi tenant workloads that run below tools like Kubernetes.
-
-## AuraeScript
-
-Aurae offers a Turing complete scripting language built on top of TypeScript called [AuraeScript](https://github.com/aurae-runtime/aurae/tree/main/auraescript). AuraeScript embeds the [Deno](https://deno.land) source code directly, and offers a remote client and SDK to interface directly with Aurae remotely. The AuraeScript library is automatically generated from the `.proto` files defined in the [Aurae Standard Library](https://aurae.io/stdlib/).
-
-Valid TypeScript files can be leveraged to replace static manifests, such as YAML, as well as interact directly with a running system.
+<!-- THE DOCUMENT -->
+
+![Deploy (aurae.io)] ![Documentation] ![Main Build (lint, compile, test)]
+
+# Summary
+
+[Aurae] is a free and [open-source] project built with the [Rust] systems
+language.
+
+The project adapts to the expanding demands of enterprise distributed systems
```

Comment:
I have very opinionated that the first sentence of the repo should either answer very concisely either:

 1. What the tool IS.
 2. What the tool DOES. 

In the case of Aurae I would want the first sentence to answer what the tool IS. 

> Aurae is a memory safe runtime daemon, process manager, and init pid 1 system designed to remotely schedule processes, containers, virtual machines, and configure node configuration such as networking an storage.

Reply:
Noted and applied accordingly.